app/src/main/java/app/nexd/android/ui/seeker/detail/SeekerDetailViewModel.kt

```diff
@@ -43,6 +43,9 @@ class SeekerDetailViewModel: ViewModel() {
                     .status(HelpRequestCreateDto.StatusEnum.DEACTIVATED)
             )
                 .observeOn(AndroidSchedulers.mainThread())
+                .doOnError {
+                    progress.value = Progress.Error
```

Comment:
this should be done in the onError consumer of subscribe instead of a side effect operation